frontend/src/app/components/work-packages/wp-breadcrumb/wp-breadcrumb-parent.component.ts

```diff
@@ -29,7 +29,6 @@
 import {Component, Input, EventEmitter, Output} from '@angular/core';
 import {WorkPackageResource} from 'core-app/modules/hal/resources/work-package-resource';
 import {WorkPackageRelationsHierarchyService} from 'core-app/components/wp-relations/wp-relations-hierarchy/wp-relations-hierarchy.service';
-import {HalResourceNotificationService} from "core-app/modules/hal/services/hal-resource-notification.service";
 import {I18nService} from 'core-app/modules/common/i18n/i18n.service';
 import {WorkPackageNotificationService} from "core-app/modules/work_packages/notifications/work-package-notification.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/c9bc013cc27eb3bb23aea9c85aaedb5b868ef858/frontend/src/app/components/work-packages/wp-breadcrumb/wp-breadcrumb-parent.component.ts#L34"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>